ladybug/windrose.py

```diff
@@ -0,0 +1,156 @@
+"""Ladybug windrose"""
+from ladybug.epw import EPW
+
+
+class WindRose(object):
+    """
+    Draws the Ladybug windrose
+    Usage:
+    """
+    __slots__ = ("epw", "HOYs", "annualHourlyData",
+                 "windCondition", "dataCondition",
+                 "numOfDirections", "centerPoint",
+                 "scale", "maxFrequency")
+
+    def __init__(self,
```

Comment:
A couple of notes:
1. We have moved from `mixedCase` to `snake_case`. All input arguments and variables in code should follow this.
2. The __init__ method should be considered for the most obvious way that a user would create the class. For other cases we should create a `classmethod`. For windrose you can create one from two DataCollections for wind speed and wind direction. If you want to create one from an epw file then you can have a `from_epw` classmethod. See WEA class as an example.
3. Conditional statements should not be part of the arguments. You should add a method to handle that.
```python
wind_rose = WindRose(wind_speed, wind_direction)
filtered_wind_rose = wind_rose.filter_by_conditional_statement(statement)
```
```